### PR TITLE
Team a

### PR DESCRIPTION
**Pull request by abidnauman**

### CODE REVIEW

**Review by lkocon69 — CHANGES_REQUESTED** [review]
This is a review of the Triple Road Train section which needs improvement - I'll need another review to approve the Traffic Analysis work for ti to eventually merge.

The task is mostly good but there are questionable samples that need to be removed before approval:

- "ab0e7665-3e5b-43d7-84d5-dee98395feb2.jpg" should be removed because it's a trailer-combo without a truck unit, which is needed for the category.
- "dist+direct+road+train+triple+wr+blog.jpg" should be removed because it's a duplicate.
- "eyad4k1cfoja1" should be removed because it's a stacked trailer that doesn't REALLY make for a triple road train, nor is there a a truck unit.
- There are multiple samples named "image#.jpg" that not only have duplicates of each other, but most are resized duplicates of higher-resolution samples that already exist.

**Comment by abidnauman** [comment]
done

**Pull request closed without merging**
